shogun-lib/src/main/java/de/terrestris/shogun/lib/model/jsonb/layer/SearchConfig.java

```diff
@@ -43,4 +44,10 @@ public class SearchConfig implements Serializable {
     )
     private String displayTemplate;
 
+    @Schema(
+        description = "The configuration for displaying search results in the result drawer.",
+        example = "{\"title\":\"{name}\",\"children\":[{\"propertyName\":\"link\",\"displayName\":\"Details\",\"fieldProps\":{\"urlDisplayValue\":\"Click here\"}}]}"
+    )
+
+    private JsonNode resultDrawerConfig;
```

Comment:
Instead of using a generic `JsonNode` we might want to create a type/class for the drawer configuration.

Reply:
Yes thanks, I have introduced types for this specific resutl drawer configuration now.